### PR TITLE
Some new documentation suggestions for the network deployment

### PR DESCRIPTION
**Pull request by SimonCommits**

### CODE REVIEW

File: system-administrators/deployment/network-deployment.md [hunk 1]
```diff
@@ -17,6 +17,15 @@ First step is to get the MedCo Deployment latest release at each node. Adapt `${
 ```bash
 export MEDCO_SETUP_DIR=~/medco MEDCO_SETUP_VER=v2.0.1
 git clone --depth 1 --branch ${MEDCO_SETUP_VER} https://github.com/ldsec/medco.git ${MEDCO_SETUP_DIR}
+
```
Comment (mickmis) [inline system-administrators/deployment/network-deployment.md:20]: This part here is docker specific and not really medco specific. So it makes sense to have it, but it should be a part of the "common problems" page.

File: system-administrators/deployment/network-deployment.md [hunk 2]
```diff
@@ -28,7 +37,11 @@ Next the _compose and configuration profiles_ must be generated using a script,
 
 ### Step 1
 
-For step 1, the network name `${MEDCO_SETUP_NETWORK_NAME}` should be common to all the nodes. `${MEDCO_SETUP_NODE_DNS_NAME}` corresponds to the machine domain name where the node is being deployed. As mentioned before the different parties should have agreed beforehand on the members of the network, and assigned an index `${MEDCO_SETUP_NODE_IDX}` to each different node to construct its UID \(starting from `0`, to `n-1`, `n` being the total number of nodes\).
+For step 1, the network name `${MEDCO_SETUP_NETWORK_NAME}` should be common to all the nodes. `${MEDCO_SETUP_NODE_DNS_NAME}` corresponds to the machine domain name where the node is being deployed. As mentioned before the different parties should have agreed beforehand on the members of the network, and assigned an index `${MEDCO_SETUP_NODE_IDX}` to each different node to construct its UID \(starting from `0`, to `n-1`, `n` being the total number of nodes\). The nodes id's need to contain 3 digits (`0` translates to `000`, `1` to `001`).
```
Comment (mickmis) [inline system-administrators/deployment/network-deployment.md:40]: The padding to 3 digits should be done automatically by the script, thus not needed to specify here. Did you have a specific issue regarding this?

**Review by mickmis — CHANGES_REQUESTED** [review]

File: system-administrators/deployment/network-deployment.md (hunk 2 above)
Comment (SimonCommits) [reply system-administrators/deployment/network-deployment.md:40]: I will confirm with Jules on that. If I remember correctly this was problematic. I'll be back with an answer soon.

Comment (SimonCommits) [reply system-administrators/deployment/network-deployment.md:40]: @Fasjul

**Pull request closed without merging**